package.json

```diff
@@ -43,7 +43,8 @@
     "tar-stream": "^1.1.1",
     "taskcluster-client": "^0.19.4",
     "url-join": "0.0.1",
-    "xml2js": "^0.4.4"
+    "xml2js": "^0.4.4",
+    "taskcluster-cli": "^0.5.3"
```

Comment:
Ok - added this to 'preinstall' in package.json and that seemed to do the trick.